test.disabled/ejabberd_tests/tests/muc_light_http_api_SUITE.erl

```diff
@@ -32,19 +32,21 @@
 %%--------------------------------------------------------------------
 
 all() ->
-
     [{group, positive}].
 
 groups() ->
-
     [{positive, [parallel], success_response()}].
 
 success_response() ->
-
     [create_unique_room,
      create_identifiable_room,
      invite_to_room,
-     send_message_to_room].
+     send_message_to_room,
+     delete_room_by_owner,
+     delete_room_by_non_owner,
```

Comment:
`delete_room_by_non_owner`, `delete_non_existent_room` and `delete_room_without_having_a_membership` should be in dedicated test group - `negative`.